lib/bolt/config/transport/docker.rb

```diff
@@ -27,6 +27,10 @@ class Docker < Base
           if @config['interpreters']
             @config['interpreters'] = normalize_interpreters(@config['interpreters'])
           end
+
+          if Bolt::Util.windows? && @config['run-as']
+            raise Bolt::ValidationError, "run-as is not supported when using PowerShell"
+          end
```

Comment:
Won't this prevent users running on Windows but connecting to a remote Docker host running Linux containers from using `run-as`?

Also, I think it might be more accurate to say "when using Windows containers" here.

Reply:
Hrm, I guess `run-as` wouldn't be available anyways since we use the Powershell shell class when running Bolt on Windows.

Reply:
Yeah, we already don't support `run-as` if you're connecting to *nix containers from Windows